src/negotiation/taker/negotiator.ts

```diff
@@ -1,23 +1,29 @@
-import { SwapRequest } from "../../cnd";
+import { SwapRequest } from "../..";
```

Comment:
🤯

Reply:
Clion was suggesting that 😅
I can also put it more explicit again.